src/ui/SymbolSelector.js

```diff
@@ -9,42 +9,42 @@ import { Select } from '@blueprintjs/select'
 import { IconNames } from '@blueprintjs/icons'
 
 class SymbolSelector extends PureComponent {
+  filterSymbol = (query, coin) => coin.toLowerCase().indexOf(query.toLowerCase()) >= 0
```

Comment:
it's a pure function so it could be even moved out of class and used as `filterSymbol` instead `this.filterSymbol`